RadixWallet/Core/Resources/Generated/AssetResource.generated.swift

```diff
@@ -113,6 +113,12 @@ internal enum AssetResource {
   internal static let unknownComponent = ImageAsset(name: "unknown-component")
   internal static let unknownDeposits = ImageAsset(name: "unknown_deposits")
   internal static let xrd = ImageAsset(name: "xrd")
+  internal static let configurationBackup = ImageAsset(name: "configuration_backup")
+  internal static let securityFactors = ImageAsset(name: "security_factors")
+  internal static let securityShields = ImageAsset(name: "security_shields")
+  internal static let shieldSetupOnboardingApply = ImageAsset(name: "shield_setup_onboarding_apply")
+  internal static let shieldSetupOnboardingBuild = ImageAsset(name: "shield_setup_onboarding_build")
+  internal static let shieldSetupOnboardingIntro = ImageAsset(name: "shield_setup_onboarding_intro")
```

Comment:
Can we put these 3 under a folder structure `ShieldBuilder/Onboarding`?
Then we will have something like this:
```
ShieldBuilder
   |_ Onboarding
       |_ shieldBuilderOnboardingApply
       |_ shieldBuilderOnboardingBuild
       |_ shieldBuilderOnboardingIntro
   |_ PrepareFactors
       |_ shieldBuilderPrepareFactorsIntro
       |_ shieldBuilderPrepareFactorsAdd
       |_ shieldBuilderPrepareFactorsCompletion
```